p2p/libp2p/directSender.go

```diff
@@ -145,6 +149,10 @@ func (ds *directSender) Send(topic string, buff []byte, peer p2p.PeerID) error {
 		return p2p.ErrMessageTooLarge
 	}
 
+	mut := ds.mutexes.Get(string(peer))
```

Comment:
rename suggestion for variable: mutexForPeer ?

Reply:
done